reference/supervisor.md

```diff
@@ -81,6 +81,7 @@ scored.
 
 WbNodeRef wb_supervisor_node_get_from_def(const char *def);
 WbNodeRef wb_supervisor_node_get_from_id(int id);
+const char *wb_supervisor_node_get_def(WbNodeRef node);
```

Comment:
Another option is to move getId and getDef functions in the same section than getTypeName.